File: Potd/05april2022.js
const countShare = (b) => {
  if (b < 4) return 0;

  let len = Math.floor((b - 2) / 2);
  console.log(len);
  const factorial = (n) => {
    if (n == 0 || n == 1) return 1;
    else return n + factorial(n - 1);
  };

  let ans = factorial(len);
  return ans;
};

console.log(countShare(5));

File: Potd/05april2022.py
import math


def countShare(B):
        #code here
  if B < 4:
    return 0
            
  input = (B - 2) / 2
  rouInput = math.trunc(input)
  print(rouInput)
  def factorial(n):
    if n == 0 or n == 1: 
      return 1
    else: 
      return n + factorial(n - 1)
        
  ans = round(factorial(rouInput))
  return ans

print(countShare(5))